### ISSUE TITLE
not able to install locally

### ISSUE THREAD

**Issue by lorvent** [REBELinBLUE/deployer-docs#1]
Hi,
I want to contribute to it but when i install it locally i am getting error

![x](http://www.clipular.com/c/5570856799961088.png?k=7VvXmviLXJNdZZXiJbi57S4VxhI)

**Comment by REBELinBLUE** [comment]
I am away from a computer this weekend but I shall take a look when I get home this evening. What OS are you using? What version of PHP and which server?

Thanks

**Comment by lorvent** [comment]
Hi,
I am using windows with xampp and php version is ```PHP 5.6.19```

**Comment by REBELinBLUE** [comment]
I don't have windows so can't really test this but from a Google search it sounds like it may be an issue with JSMin on Windows. I'll see if I can figure it out

**Comment by dasuchin** [comment]
I'm getting the same error on OS X. Wouldn't it be better to have these docs in a format that doesn't require further install to read? Just makes setting up Deployer even more cumbersome than it already is.

**Comment by lorvent** [comment]
thats also nice idea since github can display display markdown files easily

or we can make use of github pages/jekyll

**Comment by REBELinBLUE** [comment]
There might be some misunderstanding about what this repository is, it isn't supposed to be a repository which people download and run locally to view the documentation. It is the source code for the documentation website, which is currently unfinished. You're not supposed to download this repository to view the docs, they are at http://dev.phpdeployment.org/ but mostly unfinished. If you really wanted to view the documentation locally the files in https://github.com/REBELinBLUE/deployer-docs/tree/master/pages are just mark down files, but as I said it is still mostly unwritten

**Comment by lorvent** [comment]
@REBELinBLUE i mean, if we want to make changes to documentation /contribute to it, its not allowing to do.... not as an offense or to hurt you in anyway.

**Comment by dasuchin** [comment]
@REBELinBLUE fair point. My issue is that there is no link anywhere that I can find to the docs, so I assumed pulling this down to read was the only way to view them at all. Might be a good idea to add a link to them in the Deployer readme.

**Comment by REBELinBLUE** [comment]
The reason it isn't linked to is that nothing is written currently other than the install/update instructions which are already in the README, most of the pages are just lorem ipsum currently

**Comment by REBELinBLUE** [comment]
@lorvent Sorry I completely forgot about this issue, I will try and take a look at it tomorrow, although it looks like an issue in the base template so I'm not sure it is something I will be able to fix

**Comment by lorvent** [comment]
can you try gitbook.io ? its very good and has github integration.

**Comment by REBELinBLUE** [comment]
Hmm not a bad idea actually, just realised I've been reading docs with gitbook all week and it works well. Will try and take a look this evening

**Comment by REBELinBLUE** [comment]
https://rebelinblue.gitbooks.io/deployer/content/ started playing around with it
